src/main/java/machines/Truck.java

```diff
@@ -0,0 +1,11 @@
+package machines;
+
+public class Truck extends Machine {
+    public void doWork() {
+        System.out.println("Truck has started the job");
```

Comment:
The message should be more informative and indicate that it is a `Truck` that is working. Consider rephrasing it to something like 'Truck is working', to clearly reflect the type of machine.